clang/include/clang/CIR/x86.h

```diff
@@ -0,0 +1,32 @@
+//==-- x86.h - Common definitions of the x86 ABI ---------------------------==//
```

Comment:
Is this x86 only? x86_64? Please clarify in the comments. I'd prefer if the path here isolates target bits, something like `clang/include/clang/CIR/Target/<arch>.h`